### PR TITLE
Toc breadcrumb

### LINKED ISSUE #15915: Breadcrumbs are missing for most subfolders of /core/
Consider adding breadcrumbs to https://github.com/dotnet/docs/blob/master/docs/breadcrumb/toc.yml for the subfolders of /docs/core/.

### PR DESCRIPTION
**Pull request by gewarren**
- sentence-case main and core TOC's
- sentence-case breadcrumb file (used Notepad++ regex)
- add breadcrumbs for /core/ subfolders

Fixes #15915

### CODE REVIEW

**Comment by mairaw** [comment]
@gewarren do you want to take a look at the merge conflicts here?

File: docs/breadcrumb/toc.yml [hunk 1]
```diff
@@ -47,50 +47,50 @@
...
+      - name: Work with base types in .NET
         tocHref: /dotnet/standard/base-types/
         topicHref: /dotnet/standard/base-types/index
-      - name: Collections and Data Structures
+      - name: Collections and data structures
         tocHref: /dotnet/standard/collections/
         topicHref: /dotnet/standard/collections/index
         items:
-        - name: Thread-Safe Collections
+        - name: Thread-Safe collections
```
Comment (mairaw) [inline docs/breadcrumb/toc.yml]:
```suggestion
        - name: Thread-safe collections
```

**Review by mairaw — APPROVED** [review]
LGTM. Just left a small comment plus the merge conflicts need to be resolved.

**Comment by gewarren** [comment]
> LGTM. Just left a small comment plus the merge conflicts need to be resolved.

Thanks for the review!

**Pull request merged**